src/main/java/org/broadinstitute/hellbender/tools/walkers/validation/VcfInfoConcordance.java

```diff
@@ -0,0 +1,136 @@
+package org.broadinstitute.hellbender.tools.walkers.validation;
+
+import java.io.File;
+import java.util.EnumMap;
+import java.io.IOException;
+
+import org.apache.commons.collections4.Predicate;
+import org.apache.commons.lang.mutable.MutableLong;
+
+import htsjdk.variant.variantcontext.VariantContext;
+
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.BetaFeature;
+import org.broadinstitute.barclay.help.DocumentedFeature;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+
+import org.broadinstitute.hellbender.engine.ReadsContext;
+import org.broadinstitute.hellbender.engine.ReferenceContext;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.engine.AbstractConcordanceWalker;
+
+import picard.cmdline.programgroups.VariantEvaluationProgramGroup;
+
+@CommandLineProgramProperties(
+        summary=VcfInfoConcordance.USAGE_SUMMARY,
+        oneLineSummary=VcfInfoConcordance.USAGE_ONE_LINE_SUMMARY,
+        programGroup=VariantEvaluationProgramGroup.class)
+@DocumentedFeature
+@BetaFeature
+public class VcfInfoConcordance extends AbstractConcordanceWalker {
```

Comment:
Fixed, and yes I think this is different enough from concordance which evaluates a caller, because this tool is intended for filter scores or annotations.